src/Database.php

```diff
@@ -263,6 +264,26 @@ public function selectCollection($collectionName, array $options = [])
     }
 
     /**
+     * Select a GridFS bucket within this database.
+     *
+     * @see Bucket::__construct() for supported options
+     * @param string $bucketName Name of the bucket to select (defaults to 'fs')
+     * @param array  $options    Bucket constructor options
+     * @return Bucket
+     */
+    public function selectGridFSBucket($bucketName = 'fs', array $options = [])
```

Comment:
@alcaeus: Any objection if I incorporate `$bucketName` into the options array? Off hand, I don't believe we mixed optional arguments (with default values) and options arrays elsewhere in the library and I'd like to keep this consistent.